README.md

```diff
@@ -1 +1,44 @@
-# math-magician
\ No newline at end of file
+# Math Magicians
+
+> Project for getting starting with React.
+
+
+## Built With
+
+- HTML, CSS, and JavaScript
+- React
+- git/GitHub
+
+### `Getting Started
+To get a local copy up and running follow these simple example steps.
+
+  - go to the directory that you want to clone the repository in.
+  - right click and choose Git Bash here.
+  - type "git clone https://github.com/Wes-Isaac/Math-Magician.git".
+  - run "npm install" and then "npm start"
+
+## Authors
+
+👤 **Yishak**
+
+- GitHub: [Yishak ,Wes-Isaac](https://github.com/Wes-Isaac)
+- LinkedIn: [@Yishak](https://www.linkedin.com/in/yishak-wesego-b404851a7/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project is built in Microverse's second module's capstone.
```

Comment:
- [ ] OPTIONAL
Kindly place the appropriate acknowledgement section of your ReadMe.